### ISSUE TITLE
Introduce monochrome mode

### ISSUE THREAD

**Issue by alekslovestech** [alekslovestech/chromatic-circle#186]
Introduce a toggle that goes between a standard piano coloring with black and white keys, to all keys being white (circular keyboard only)
